opencraft/settings.py

```diff
@@ -77,7 +77,8 @@
     'pr_watch',
     'userprofile',
     'registration',
-    'backup_swift'
+    'backup_swift',
+    'community',
```

Comment:
@antoviaque Oh, do you filter out GitHub replies that don't mention you too? I assumed you just did that on JIRA.

> This could be mocked up, no?

If we have a .yml file in the test suite, it cannot contain a part that is randomly generated. So yeah, we can add some messy hacks/mocks to make the test suite work and successfully compare to a .yml file, but we won't be able to provide the community with a link that includes randomly generated secure vars just for them. The only way to do that would be a view. Of course, nobody should be using our .yml file without changing the vars anyways, but it's better if we can provide an option that's secure by default.

I don't consider any of these things a blocker though. I'm fine with changing to the approach you're suggesting. I just think it's a bit less nice :)